packages/itwin/tree-widget/src/components/trees/VisibilityTreeRenderer.tsx

```diff
@@ -14,13 +14,44 @@ import type { NodeCheckboxRenderProps } from "@itwin/core-react";
 import type { IPresentationTreeDataProvider } from "@itwin/presentation-components";
 import type { VisibilityTreeFilterInfo } from "./Common";
 
+/**
+ * Props for visibility tree node renderer.
+ * @public
+ */
+export interface visibilityTreeRendererProps {
+  /**
+   * Specifies whether the icon at the left of the node label should be rendered.
+   */
+  iconsEnabled: boolean;
+  /**
+   * Specifies whether node description should be enabled.
+   */
+  descriptionEnabled: boolean;
+  /**
+   * Defines the size in pixels of how much the node label should be pushed to the right from the checkbox.
+   * Default value is 20.
+   */
+  levelOffset?: number;
+  /**
+   * Defines the size in pixels of how much the leaf node label should be pushed to the right from the checkbox.
+   * @note This value applies only to the leaf nodes.
+   * Default value is 24.
+   */
+  expansionToggleWidth?: number;
```

Comment:
name of the attribute doesn't really match the docs - are they ok?

Reply:
fixed